org-roam.el

```diff
@@ -594,6 +602,40 @@ specified via the #+ROAM_ALIAS property."
            (slug (-reduce-from #'replace (strip-nonspacing-marks title) pairs)))
       (s-downcase slug))))
 
+;;;; Completion
+(defun org-roam---helm-candidate-transformer (candidates source)
+  (let ((prefixed-pattern (propertize
+                           " " 'display
+                           (propertize "[?]" 'face 'helm-ff-prefix))))
+    (cons (concat prefixed-pattern " " helm-pattern)
+          candidates)))
+
+(defun org-roam--completing-read (prompt choices &optional require-match initial-input)
+  "Present a PROMPT with CHOICES and optional INITIAL-INPUT.
+If REQUIRE-MATCH is `t', the user must select one of the CHOICES.
+Return user choice."
+  (cond
+   ((eq org-roam-completion-system 'default)
+    (completing-read prompt choices nil require-match initial-input))
+   ((eq org-roam-completion-system 'helm)
+    (unless (and (fboundp 'helm)
+                 (fboundp 'helm-build-sync-source))
+      (user-error "Please install helm from \
+https://github.com/emacs-helm/helm"))
+    (let* ((source (helm-build-sync-source "Title"
```

Comment:
should this be prompt instead of "Title"?

Reply:
Yes, it should!